Neos.ContentRepository/Classes/Domain/Utility/NodeContexts.php

```diff
@@ -20,12 +20,12 @@
     /**
      * Determine if the given node path is a context path.
      *
-     * @param string $string path or identifier
+     * @param string $string context path or context identifier
      * @return boolean
      */
     public static function hasContext($string)
     {
-        return (strpos($string, '@') !== false);
+        return (preg_match('/@/u', $string) === 1);
```

Comment:
Okay 😉 

So should we use a match for `MATCH_PATTERN_CONTEXTPATH` or `MATCH_PATTERN_CONTEXTIDENTIFIER` to make this more explicit?